lldb/source/API/SBTarget.cpp

```diff
@@ -198,16 +198,24 @@ SBDebugger SBTarget::GetDebugger() const {
 }
 
 SBStructuredData SBTarget::GetStatistics() {
+   LLDB_INSTRUMENT_VA(this);
+  SBStatisticsOptions options;
+  options.SetSummaryOnly(false);
```

Comment:
This defaults to false right? I would make sure the default `SBStatisticsOptions` constructor sets everything to good defaults and we can remove this line.